src/app/app-routing.module.ts

```diff
@@ -1,20 +1,31 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
-import { HomeComponent } from './pages/home/home.component';
+import { AuthRoutingModule } from './auth/auth.routing';
+import { PagesComponent } from './pages/pages.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7127387)
```suggestion
import { PagesComponent } from "./pages/pages.component";
```